pkg/operator/operator.go

```diff
@@ -121,23 +127,24 @@ func NewOperator() (context.Context, *Operator) {
 	})
 
 	// Client Config
-	config := controllerruntime.GetConfigOrDie()
+	config := ctrl.GetConfigOrDie()
 	config.RateLimiter = flowcontrol.NewTokenBucketRateLimiter(float32(options.FromContext(ctx).KubeClientQPS), options.FromContext(ctx).KubeClientBurst)
 	config.UserAgent = fmt.Sprintf("%s/%s", appName, Version)
 
 	// Client
 	kubernetesInterface := kubernetes.NewForConfigOrDie(config)
 
 	// Logging
-	logger := logging.NewLogger(ctx, component)
-	ctx = knativelogging.WithLogger(ctx, logger)
-	logging.ConfigureGlobalLoggers(ctx)
+	logger := log.FromContext(ctx).WithName(component)
```

Comment:
So, just reading this: It seems like we should run L140 first and L141 first and then add the name to the zapr logger?

So maybe something like

```go
logger := zapr.New().WithName(component)
log.SetLogger(logger)
klog.SetLogger(logger)

```

Reply:
From there, I think everything just works